packages/e2e-tests/scripts/get-stats.ts

```diff
@@ -147,14 +149,6 @@ export function getStats() {
       return a[0].localeCompare(b[0]);
     }
   });
-  entries.forEach(([key, { recordingId, runtime, runtimeOS, runtimeReleaseDate }]) => {
```

Comment:
I'm re-adding this.

Reply:
I see what you are doing there... But now `runtimeReleaseDate` still has the wrong type 👀

Reply:
Then change the type? I didn't see any type errors locally or on CI, which maybe means that TS isn't checking this directory?

Reply:
Alternately you could change the print stats script to do this stringification.

Reply:
1. TSC doesn't complain, because you cast it to `any`.
2. Your change breaks our assumption that this is `Date` ([src](https://github.com/replayio/devtools/blob/b30146feafcd1f0acbe92331b94ad298207a7334/packages/e2e-tests/scripts/buildkite_run_fe_tests.ts#L27)):
   `exampleFileInfo.runtimeReleaseDate.getUTCFullYear() === 2024`

Reply:
I'm moving it to stringification. For future reference, I'd strongly advise against coercing types into something they are not, since this means that we cannot trust the defined types.

Reply:
Where is the code that does the printing?

Reply:
(temporarily removing this bit, so I can run tests)

Reply:
Fixed `print-test-stats`:
![image](https://github.com/replayio/devtools/assets/282899/63277c3a-c971-4018-86c5-a26142dacc44)

Reply:
For future reference, I'd strongly advise against coercing types into something they are not, since this means that we cannot trust the defined types.

Reply:
I had forgotten that the `any` coercion was even there. I just reverted the changed lines because I didn't want the output to change, but I should have just left a PR comment and let you address it how you saw fit. Thanks for updating the print script.

Reply:
This change means the `sortedTestFileToInfoMap` variable declared above (line 137) is no longer referenced.

Reply:
Fixing it now.
I just learned that `npm run lint` does not cover this directory (or we allow undeclared vars). Will pay more attention 😊